lib/api/multipartDelete.js

```diff
@@ -51,6 +51,8 @@ function multipartDelete(authInfo, request, log, callback) {
                     keys: [objectKey],
                     byteLength: partSizeSum,
                     location,
+                    numberOfObjects: 1,
```

Comment:
We don't have to set `numberOfObjects` here as it's not counted until mpu is finished